console/src/main/java/org/georchestra/console/ws/passwordrecovery/PasswordRecoveryFormController.java

```diff
@@ -161,7 +165,7 @@ public String generateToken(
 			this.userTokenDao.insertToken(account.getUid(), token);
 
 			String contextPath = this.config.getPublicContextPath();
-			String url = makeChangePasswordURL(request.getServerName(), request.getServerPort(), contextPath, token);
+			String url = makeChangePasswordURL(this.georConfig.getProperty("publicUrl"), contextPath, token);
```

Comment:
This will deprecate the need to customize connector parameters `proxyName` & `proxyPort` as per https://github.com/georchestra/georchestra/blob/17.12/docs/setup/tomcat.md#configure-connector => update the doc ?

Reply:
yes !

Reply:
Done with : 3e43ce726e4fb685bca3ddc9781e61c86dfccad5